Orange/widgets/visualize/owpythagorastree.py

```diff
@@ -347,6 +355,11 @@ def commit(self):
         data = self.tree_adapter.get_instances_in_nodes(
             self.clf_dataset, [item.tree_node.label for item in items])
         self.send('Selected Data', data)
+        nodes = [i.tree_node.label for i in self.scene.selectedItems()
```

Comment:
Isn't this already computed above, as the second argument in the call of `get_instances_in_nodes`?